modules/tinymce/src/core/test/ts/browser/EditorManagerTest.ts

```diff
@@ -1,7 +1,12 @@
 import { after, afterEach, before, describe, it } from '@ephox/bedrock-client';
+import { Type } from '@ephox/katamari';
 import { Remove, Selectors } from '@ephox/sugar';
 import { assert } from 'chai';
 import 'tinymce';
+// eslint-disable-next-line @tinymce/no-main-module-imports
```

Comment:
Why was this needed?

Reply:
The test doesn't run in isolation otherwise. Since we are not pulling in `mcagar`, we don't get the automatic import of tinymce, theme and model

Reply:
It should be lazy loaded then maybe you need to set the base url. Check other tests.
This would make it actually import all of the silver theme in the test bundle.

Reply:
Thanks, I'll have a look 👍

Reply:
Hmm, the baseUrl is already set here: https://github.com/tinymce/tinymce/pull/10277/files#diff-9f3c744ede87f4e7e0529225454f688bc0a477c8c9ac3d6304a8153cb733939cR29

And I tried using what mcagar does.

Maybe I am missing something

Reply:
This test is a bit weird because it wasn't really designed to load a full editor. But perhaps it should, since it clearly depends on it when it runs as a part of a wider test suite.

You might be able to get away with importing the actual `dom/Model` and `silver/Theme` files rather than using their `Main` imports to avoid the eslint warnings 🤔

Reply:
Hmm, I pulled down the branch and ran the `pageUid/editorUid` in isolation just fine? What test didn't work in isolation?